packages/odata-common/src/time.ts

```diff
@@ -1,3 +1,15 @@
+/**
+ * @internal
+ */
+export const durationRegex_v2 =
```

Comment:
Please make sure to comply with our [STYLEGUIDE](https://github.com/SAP/cloud-sdk-js/blob/2.0/STYLEGUIDE.md)

Reply:
Also, minor: I think in v2, there is no "duration" in EDM, I think it is called "time" because it can be used for both, durations and time of day.

Reply:
The original variable was "duration" so I kept it same. Also I thought the format PT..... is generally to denote duration (though we also use it for time) so maybe would be okay.